xapi-nbd.opam

```diff
@@ -4,27 +4,18 @@ authors: ["dave.scott@citrix.com"]
 homepage: "https://github.com/xapi-project/xapi-nbd"
 bug-reports: "https://github.com/xapi-project/xapi-nbd/issues"
 dev-repo: "https://github.com/xapi-project/xapi-nbd.git"
-build: [
-  ["oasis" "setup"]
-  ["./configure" "--prefix" prefix]
-  [make]
-]
-install: [
-  [make "install" "BINDIR=%{bin}%"]
-]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
 remove: [
```

Comment:
The remove instruction should be removed - I guess it's not necessary with a generated install file.